### PR TITLE
kill: Fix kill tests

### PR DESCRIPTION
**Pull request by jcvenegas**
runc was failing with our kill test.

- Only SIGKILL and SIGSTOP can not be caught, blocked or ignored, skip
  signals that are not forwared by cc-shim.

- Show the signal name when running the tests (not only number).

Fixes: #640

Signed-off-by: Jose Carlos Venegas Munoz <jose.carlos.venegas.munoz@intel.com>

### CODE REVIEW

**Comment by jcvenegas** [comment]
Running this locally fails to me using cc-runtime.
```
make integration RUNTIME=cc-runtime FOCUS="docker kill"
Summarizing 7 Failures:

[Fail] docker kill killing container [It] with '3'(quit) signal 
/home/jcvenega/go/src/github.com/clearcontainers/tests/integration/docker/kill_test.go:96

[Fail] docker kill killing container [It] with '4'(illegal instruction) signal 
/home/jcvenega/go/src/github.com/clearcontainers/tests/integration/docker/kill_test.go:96

[Fail] docker kill killing container [It] with '7'(bus error) signal 
/home/jcvenega/go/src/github.com/clearcontainers/tests/integration/docker/kill_test.go:96

[Fail] docker kill killing container [It] with '8'(floating point exception) signal 
/home/jcvenega/go/src/github.com/clearcontainers/tests/integration/docker/kill_test.go:96

[Fail] docker kill killing container [It] with '11'(segmentation fault) signal 
/home/jcvenega/go/src/github.com/clearcontainers/tests/integration/docker/kill_test.go:96

[Fail] docker kill killing container [It] with '13'(broken pipe) signal 
/home/jcvenega/go/src/github.com/clearcontainers/tests/integration/docker/kill_test.go:96

[Fail] docker kill killing container [It] with '28'(window changed) signal 
/home/jcvenega/go/src/github.com/clearcontainers/tests/integration/docker/kill_test.go:96

Ran 31 of 87 Specs in 383.241 seconds
FAIL! -- 24 Passed | 7 Failed | 0 Pending | 56 Skipped --- FAIL: TestIntegration (401.68s)
FAIL

```

**Comment by jcvenegas** [comment]
@devimc  could you take a look to this PR?

File: integration/docker/kill_test.go [hunk 1]
```diff
@@ -96,23 +97,29 @@ var _ = Describe("docker kill", func() {
...
+		// This is not forwarded by cc-shim https://github.com/clearcontainers/runtime/issues/769
+		//withSignal(syscall.SIGQUIT, canBeTrapped),
+		// This is not forwarded by cc-shim https://github.com/clearcontainers/runtime/issues/769
+		// withSignal(syscall.SIGILL, canBeTrapped),
 		withSignal(syscall.SIGTRAP, canBeTrapped),
 		withSignal(syscall.SIGIOT, canBeTrapped),
-		withSignal(syscall.SIGBUS, cannotBeTrapped),  //135
-		withSignal(syscall.SIGFPE, cannotBeTrapped),  //136
+		// This is not forwarded by cc-shim https://github.com/clearcontainers/runtime/issues/769
+		// withSignal(syscall.SIGBUS, canBeTrapped)
```
Comment (devimc) [inline integration/docker/kill_test.go]: can you skip them? I'd like to see what signals are skipped

Comment (jcvenegas) [reply integration/docker/kill_test.go]: @devimc  test are skipped.

File: integration/docker/kill_test.go [hunk 2]
```diff
@@ -68,6 +69,15 @@ var _ = Describe("docker kill", func() {
 		func(signal syscall.Signal, expectedExitCode int) {
 			args = []string{"--name", id, "-dt", Image, "sh", "-c"}
 
+			switch signal {
+			case syscall.SIGQUIT, syscall.SIGILL, syscall.SIGBUS, syscall.SIGFPE, syscall.SIGSEGV, syscall.SIGPIPE:
+				Skip("This is not forwarded by cc-shim " +
+					"https://github.com/clearcontainers/runtime/issues/769")
+			case syscall.SIGWINCH:
```
Comment (sboeuf) [inline integration/docker/kill_test.go]: Why this is not forwarded ? I am pretty sure it is by the shim.

**Pull request merged**